tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -18,22 +18,22 @@ set(SOURCES
   Ecs/SystemSchedulerTest.cpp
   Ecs/SparseSetMemoryContainerTest.cpp
 
-  Interop/NovelRTInteropUtilsTest.cpp
-  Interop/Ecs/NrtCatalogueTest.cpp
-  Interop/Ecs/NrtComponentBufferMemoryContainerTest.cpp
-  Interop/Ecs/NrtComponentCacheTest.cpp
-  Interop/Ecs/NrtEntityCacheTest.cpp
-  Interop/Ecs/NrtSparseSetMemoryContainerTest.cpp
-  Interop/Ecs/NrtSystemSchedulerTest.cpp
-  Interop/Maths/GeoBoundsTest.cpp
-  Interop/Maths/GeoMatrix4x4Test.cpp
-  Interop/Maths/GeoVector2FTest.cpp
-  Interop/Maths/GeoVector3FTest.cpp
-  Interop/Maths/GeoVector4FTest.cpp
-  Interop/Maths/QuadTreePointTest.cpp
-  Interop/Maths/QuadTreeTest.cpp
-  Interop/SceneGraph/NovelRTSceneNodeTest.cpp
-  Interop/Timing/NovelRTTimestampTest.cpp
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/NovelRTInteropUtilsTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Ecs/NrtCatalogueTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Ecs/NrtComponentBufferMemoryContainerTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Ecs/NrtComponentCacheTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Ecs/NrtEntityCacheTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Ecs/NrtSparseSetMemoryContainerTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Ecs/NrtSystemSchedulerTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Maths/GeoBoundsTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Maths/GeoMatrix4x4Test.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Maths/GeoVector2FTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Maths/GeoVector3FTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Maths/GeoVector4FTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Maths/QuadTreePointTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Maths/QuadTreeTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/SceneGraph/NovelRTSceneNodeTest.cpp>
+  $<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop/Timing/NovelRTTimestampTest.cpp>
```

Comment:
I'm wondering if there's a better way to do this. It's not a blocker, but maybe just like, "if interop, append these files" type deal? Not sure how doable or nice it would look, just musing.

Reply:
That was essentially what my previous change had no?... It requires a separate `if` block, whereas this is inlined with the same block the rest of the test sources use (based on @FiniteReality's suggestion).